### PR TITLE
add ga profile id to advanced page

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@smn @codiebeulaine ready for review

File: unicoremc/forms.py [hunk 1]
```diff
@@ -1,19 +1,18 @@
...
-
-    def __init__(self, *args, **kwargs):
-        super(ProjectForm, self).__init__(*args, **kwargs)
-
-        self.fields["available_languages"].widget = CheckboxSelectMultiple()
-        self.fields[
-            "available_languages"].queryset = Localisation.objects.all()
-        self.fields["default_language"].widget = RadioSelect()
-        self.fields["default_language"].choices = [
-            (l.pk, str(l)) for l in Localisation.objects.all()]
```
Comment (smn) [inline unicoremc/forms.py:19]: Why did we go this round about way earlier?

**Comment by smn** [comment]
:+1:

File: unicoremc/forms.py (hunk 1 above)
Comment (miltontony) [inline unicoremc/forms.py:19]: My initial reading on changing the queryset yielded to examples where people changed it in  `__init__`. When I was adding the new field today, it didn't feel right anymore so I went into the django docs again to see if queryset is overridable on a Field level and that's when I found the `ModelChoiceField` and the `ModelMultipleChoiceField`. All others don't have the `queryset` param.

Comment (smn) [inline unicoremc/forms.py:19]: thanks!

**Comment by codiebeulaine** [comment]
:+1:  Quick question Milton if you have time, the file you added to migrations, did you write all of that from scratch or?

**Pull request merged**